vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/per_hour_frequency_cap.cc

```diff
@@ -5,7 +5,7 @@
 
 #include "bat/ads/internal/frequency_capping/exclusion_rules/per_hour_frequency_cap.h"
 #include "bat/ads/internal/frequency_capping/frequency_capping.h"
-#include "bat/ads/internal/time.h"
+#include "bat/ads/internal/time_util.h"
```

Comment:
We don't seem to be using anything defined in `time_util.h`, is that right? Are we including it just to get `base/time/time.h`? I see a bunch of similar includes of `time_util.h`.